crates/sui-protocol-config/src/lib.rs

```diff
@@ -198,6 +198,7 @@ const MAX_PROTOCOL_VERSION: u64 = 69;
 // Version 69: Sets number of rounds allowed for fastpath voting in consensus.
 //             Enable smart ancestor selection in devnet.
 //             Enable G1Uncompressed group in testnet.
+//             Enable probing for accepted rounds in round prober.
```

Comment:
Once Tasos lands his PR with Protocol Version 70 I will rebase this PR and move this feature to that version